components/micro-gateway-cli/src/main/resources/templates/tokenServices.mustache

```diff
@@ -123,8 +123,14 @@ service tokenService on tokenListenerEndpoint, apiSecureListener {
     resource function tokenResource(http:Caller caller, http:Request req) {
         gateway:checkExpectHeaderPresent(req);
         http:Client tokenEndpointClient = gateway:getTokenEndpoint();
-        var response = tokenEndpointClient->forward(gateway:getConfigValue(gateway:KM_CONF_INSTANCE_ID, gateway:KM_TOKEN_CONTEXT, gateway:DEFAULT_KM_TOKEN_CONTEXT) +
-                 <@untainted>req.rawPath, req);
+        string tokenContext = gateway:getConfigValue(gateway:KM_CONF_INSTANCE_ID, gateway:KM_TOKEN_CONTEXT, gateway:DEFAULT_KM_TOKEN_CONTEXT);
+        if (!tokenContext.startsWith("/")) {
+            tokenContext = "/" + tokenContext;
+        }
+        if (tokenContext.endsWith("/")) {
+            tokenContext = tokenContext.substring(0, tokenContext.length() - 1);
+        }
+        var response = tokenEndpointClient->forward(tokenContext + <@untainted>req.rawPath, req);
```

Comment:
I think we need to fix this for other proxy resources like /authorizwe, /revoke and /userInfo as well

Reply:
fixed with https://github.com/wso2/product-microgateway/pull/1211/commits/106ee70e453b5af983b0693e76a92c61f2177154